controls/radrichtextbox/features/positioning.md

```diff
@@ -9,72 +9,145 @@ published: True
 
 # Positioning
 
-
-
 The positioning feature in the __RadRichTextBox__ is used to navigate through document's content and to get information about the document's elements at a specific position. The __RadDocument__ uses the positioning to track the movement of the caret and to control the selection.

```

Comment:
Can we move the TOC just after this paragraph and separate the rest of this section in a new (Overview) section?